src/containers/InteractiveMessage/InteractiveOptions/InteractiveOptions.module.css

```diff
@@ -53,3 +53,12 @@
   composes: Translation from 'App.module.css';
   margin-bottom: 8px;
 }
+
+@media (max-width:768px) {
+  .RadioLabelWrapper:last-child {
+    margin-left: 0px;
+  }
+  .ForMobile {
```

Comment:
It's good to have className reflecting where it's used etc. So in this case may be chang it to `RadioGroup`